sssom/cli.py

```diff
@@ -501,5 +502,20 @@ def reconcile_prefixes(input: str, reconcile_prefix_file: Path, output: TextIO):
     write_table(recon_msdf, output)
 
 
+@main.command()
+@input_argument
+@output_option
+def sort_columns(input: str, output: TextIO):
```

Comment:
Can we make a generic `sort` method that takes as an input two flags `--sort-columns` and `--sort-rows` (both true by default)?

Reply:
Done!

Reply:
I cannot redefine `sort` since it throws errors in `flake8` and `mypy` : 
```
Name "sort" already defined (possibly by an import)
F811 redefinition of unused 'sort' from line 32
```